runtime/oti/j9java8packages.gperf

```diff
@@ -0,0 +1,470 @@
+/*******************************************************************************
+ * Copyright (c) 2017, 2017 IBM Corp. and others
+ *
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/
+ * or the Apache License, Version 2.0 which accompanies this distribution and
+ * is available at https://www.apache.org/licenses/LICENSE-2.0.
+ *
+ * This Source Code may also be made available under the following
+ * Secondary Licenses when the conditions for such availability set
+ * forth in the Eclipse Public License, v. 2.0 are satisfied: GNU
+ * General Public License, version 2 with the GNU Classpath
+ * Exception [1] and GNU General Public License, version 2 with the
+ * OpenJDK Assembly Exception [2].
+ *
+ * [1] https://www.gnu.org/software/classpath/license.html
+ * [2] http://openjdk.java.net/legal/assembly-exception.html
+ *
+ * SPDX-License-Identifier: EPL-2.0 OR Apache-2.0
+ *******************************************************************************/
+ 
+/*
+ * This file is the input to the gperf utility for generating a perfect
+ * hash table of attribute names, which is used by attributeTagFor().
+ *
+ * To generate attrlookup.h, use the following command-line:
+ *
+ *   gperf -CD -t --output-file=j9java8packages.h j9java8packages.gperf
+ *
+ * (The gperf utility is a perfect hash function generator that is readily available on Linux.)
+ *
+ */
+
+%language=ANSI-C
+%define hash-function-name packageNameHash
+%define lookup-function-name lookupJava8Package
+
+struct AttributeType
+{
+	const char *name;
+};
+
+%%
+com/ibm/dataaccess
+com/ibm/gpu
+com/ibm/jit/crypto
+com/ibm/jit
+com/ibm/jvm/io
+com/ibm/jvm
+com/ibm/lang/management
+com/ibm/le/conditionhandling
+com/ibm/oti/lang
+com/ibm/oti/reflect
+com/ibm/oti/shared
+com/ibm/oti/util
+com/ibm/oti/vm
+com/ibm/tools/attach/javaSE
```

Comment:
attach/javaSE, attach/spi, attach don't exist any more

Reply:
rather com/sun/tools/attach and attach/spi should be added

Reply:
I couldn't find `com/sun/tools/attach` and `com/sun/tools/attach/spi` in JDK8. I checked jdk8u151-b12

Reply:
These classes are in tools.jar prior to Java 9.

Reply:
Thanks, I was looking in the src.zip which doesn't actually contain all the source